src/providers/cross-chain-sync.provider.ts

```diff
@@ -97,161 +89,43 @@ export const crossChainSyncProvider = () => {
     return omniEscrowLocks.value?.remoteUser;
   });
 
-  /**
-   * votingEscrowLocks contains the user's original veBAL data
-   * slope and bias is how a user's "balance" is stored on the smart contract
-   */
-  const {
-    data: mainnetVotingEscrowResponse,
-    isInitialLoading: isLoadingVotingEscrow,
-    refetch: refetchVotingEscrow,
-  } = useVotingEscrowLocksQuery(Network.MAINNET, account);
-
-  const {
-    data: arbitrumVotingEscrowResponse,
-    refetch: refetchVotingEscrowArbitrum,
-  } = useVotingEscrowLocksQuery(Network.ARBITRUM, remoteUser);
-
-  const {
-    data: polygonVotingEscrowResponse,
-    refetch: refetchVotingEscrowPolygon,
-  } = useVotingEscrowLocksQuery(Network.POLYGON, remoteUser);
+  const mainnetCrossChainNetwork = useCrossChainNetwork(
+    Network.MAINNET,
+    account
+  );
 
-  const {
-    data: optimismVotingEscrowResponse,
-    refetch: refetchVotingEscrowOptimism,
-  } = useVotingEscrowLocksQuery(Network.OPTIMISM, remoteUser);
+  const mainnetEscrowLocks = computed(
+    () => mainnetCrossChainNetwork.votingEscrowLocks.value
+  );
 
-  const {
-    data: gnosisVotingEscrowResponse,
-    refetch: refetchVotingEscrowGnosis,
-  } = useVotingEscrowLocksQuery(Network.GNOSIS, remoteUser);
+  type UseCrossChainNetworkResponse = Record<
```

Comment:
may be better to extract type outside composable?

Reply:
Ok, maybe you can also move some other types from the initial file.